pkg/etcd/etcd_int_test.go

```diff
@@ -9,76 +9,76 @@ import (
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
-	"github.com/rancher/k3s/pkg/util/tests"
+	"github.com/rancher/k3s/tests/testutil"
```

Comment:
Not a blocker for merging: This reminds me, this package name should be changed to remove the redundancy. I thought I had commented that but I guess I didn't. We can use an import alias to disambiguate if necessary.

Reply:
Commented below at the name change.

Reply:
Fixed, added alias to differentiate from other util packages.